README.md

```diff
@@ -2,7 +2,7 @@
 
 # [C a d H u b](https://cadhub.xyz)
 
-[![Netlify Status](https://api.netlify.com/api/v1/badges/77f37543-e54a-4723-8136-157c0221ec27/deploy-status)](https://app.netlify.com/sites/cadhubxyz/deploys)
+<!-- [![Netlify Status](https://api.netlify.com/api/v1/badges/77f37543-e54a-4723-8136-157c0221ec27/deploy-status)](https://app.netlify.com/sites/cadhubxyz/deploys) -->
```

Comment:
Removing because more than often it displays as failed, when what really happened is build was cancel because there was a change to the docs not the app so the build finished early. Not sure if this can be sorted out or not.